docs/development/getting_started_locally.md

```diff
@@ -8,7 +8,7 @@ In this guide, we will start a [KinD](https://kind.sigs.k8s.io/) cluster which i
 
 The Gardener components, however, will be run as regular processes on your machine (hence, no container images are being built).
 
-![Architecture Diagram](content/getting_started_locally.png)
+![Architecture Diagram](https://raw.githubusercontent.com/gardener/gardener/master/docs/development/content/getting_started_locally.png)
```

Comment:
From markdown's point of view the current revision in the HEAD is a valid and working link. It may be rather a docforge issue (cc @dimitar-kostadinov) or issue with the onboarding side (cc @ashwani2k). Hence, I don't think we have to change something on this guide.

Reply:
Basically, absolute links pointing to the same repository should be avoided as they interfere with the documentation build for a specific version. The same image appears properly on: https://gardener.cloud/docs/gardener/development/getting_started_locally/